raiden_contracts/tests/test_deploy_script.py

```diff
@@ -1023,3 +1027,25 @@ def test_verify_monitoring_service_deployment_with_wrong_onchain_token_address()
             service_registry_address=EMPTY_ADDRESS,
             user_deposit_address=FAKE_ADDRESS,
         )
+
+
+def test_user_deposit_deployment_with_wrong_one_to_n_address() -> None:
+    """ ContractVerifier.verify_user_deposit_deployment raises on a wrong OneToN address """
+    user_deposit_constructor_arguments = [
+        "0xDa12Dc74D2d0881749CCd9330ac4f0aecda5686a",
+        UINT256_MAX,
+    ]
+    wrong_one_to_n_address = FAKE_ADDRESS
+    user_deposit_mock = MagicMock()
+    mock_token_address = MagicMock()
+    mock_token_address.call.return_value = user_deposit_constructor_arguments[0]
+    user_deposit_mock.functions.token.return_value = mock_token_address
+    with pytest.raises(RuntimeError):
```

Comment:
As a general pointer, I always find it helpful to separate test setup logic (prepping vars, mocks etc) by a new line from the remainaing test code.